Automattic/woocommerce-payments
# Update payment intent in UPE workflow

Fixes #1830 

#### Changes proposed in this Pull Request

Updates the payment intend created at the beginning of the UPE checkout flow with the final amount of the order and the customer id.
It may be desired to add the `level3` data at this point but I am not familiar enough with it to know if it's necessary.

#### Testing instructions

See https://github.com/Automattic/woocommerce-payments/pull/1890 for full instructions.
The only thing different here is to ensure the transaction total shown on stripe dashboard matches the order total. Switch the shipping method chosen to change the order total during checkout.

-------------------
